lib/generators/rolemodel/modals/templates/app/views/layouts/modal.html.slim.tt

```diff
@@ -9,12 +9,11 @@ html
   body
     = turbo_frame_tag 'modal' do
       .modal-wrapper(
-        class=class_names('modal-wrapper--active' => response.message == 'Unprocessable Entity')
+        class=class_names('modal-wrapper--active' => response.message.starts_with?('Unprocessable'))
         data-controller="toggle"
         data-toggle-perform-on-connect-value=(response.message == 'OK')
         data-toggle-active-class="modal-wrapper--active"
-        data-turbo-cache="false"
-        data-action="keydown.esc@window->toggle#off"
```

Comment:
I think this is a bad pattern, as it leads to accidental data-entry loss.  It also doesn't play nicely with browser auto-complete features.